source/content/oss-support-levels.md

```diff
@@ -0,0 +1,42 @@
+tags: [collaborate]
+contributors: [greg-1-anderson, kporras07]
+reviewed: "2022-03-02"
+---
+
+At Pantheon, we appreciate open source contributions. In an effort to effectively maintain our various public repositories in [GitHub](https://github.com/orgs/pantheon-systems/repositories), we tag Pantheon repositories with support level badges. This page describes each support level and details the support expectations for repositories.
+
+## Early Access Support
+
+A project in Pilot support level is a new product or feature set that we are working on. The code is publicly available, but if there are any accompaning platform components you will need an invite through the project pilot program. Even though we make our best effort, there may be bugs and we need your help to identify them so that we can fix them. The product or feature set may not be feature complete and development is still going on. Support in this stage is provided by the Engineering team, and is only available to members of the program. For more information on 
+
+## Limited Availability Support
+
+A project in this support level is a product or feature set that is complete and can be used in all environments including production. Clients can use the new functionality without requiring an invite but it is an opt-in process. You are highly encouraged to report any bug you may find so that we can make a better product for everyone. Support in this stage is available through regular Pantheon channels.
+
+## Actively Maintained Support
+
+A project in this support level has been publicly launched and has several eyes over it ensuring that everything works as expected. Anyone could use this project and create issues and Pull Requests in the repository as needed and we will respond to them in a timely manner. Support in this stage is available through regular Pantheon channels.
+
+## Minimal Support
+
+A project in this support level rarely receives updates but if there is an important bug or security update we will make our best effort to address it in a timely manner.
+
+## Unofficial Support
+
+A project that has unofficial support level is usually a Pantheon employee initiative. It typically does not have resources allocated for maintenance. Support is not guaranteed for the project because it depends on the availability and afffiliation of the project maintainer.
+
+## Unsupported 
+
+An unsupported project is typically only for internal use at Pantheon. Although the source code is available, use these projects at your own risk, as support is not available.
+
+## Deprecated 
+
+A deprecated project is a project that has been retired or replaced by a code base that is no longer supported by Pantheon. Details about the project can be found in the `README.md` or you could try asking us in the [Community Slack](https://slackin.pantheon.io/) if you are in a need for the replacement or guidance about where to go. Using a project that has been deprecated is not encouraged. 
```

Comment:
"replaced by a code base that is no longer supported by Pantheon" no longer modifies the correct object. The deprecated project is no longer supported by Pantheon; the code base that replaces it probably is.

Maybe: "replaced by some other project, and is no longer supported by Pantheon."